src/components/GameplayContainer.js

```diff
@@ -9,23 +9,32 @@ import * as scoreActions from "../actions/scoreActions";
 
 class GameplayContainer extends React.Component {
   componentDidMount() {
-    this.timer = setInterval(() => this.props.actions.beatmapActions.incrementBeat(), 1000);
+    this.timer = setInterval(
+      () => this.props.actions.beatmapActions.incrementBeat(),
+      1000
+    );
   }
 
   render() {
     // render multiple circles here
     return (
       <div>
         <div> Score: {this.props.score.currentScore}</div>
-        <div> Accuracy: {this.props.score.notesHit/this.props.score.notesTotal*100}%</div>
+        <div>
+          {" "}
```

Comment:
@niccalle what does this do

Reply:
@minho814 idk prettier-atom automatically formats the file when you touch it. I'm sure we'll change this stuff soon enough